release-notes/opensearch-dashboards.release-notes-2.0.0-rc1.md

```diff
@@ -0,0 +1,98 @@
+* [CVE-2021-44906] Bump minimist from 1.2.5 to 1.2.6 ([#1377](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1377))
+* [CVE] Bumps node-forge from v1.2.1 to v1.3.0 ([#1369](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1369))
+* [CVE] Bump lodash-es from 4.17.15 to 4.17.21 ([#1343](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1343))
+* [CVE] Resolves ansi-regex to v5.0.1 ([#1320](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1320))
+* [CVE] Bump url-parse from 1.5.3 to 1.5.7 ([#1257](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1257))
+* [CVE-2022-0536] Bump follow-redirects from 1.14.7 to 1.14.8 ([#1247](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1247))
+* [CVE] Upgrades node-forge from v0.10.0 to v1.2.1 ([#1239](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1239))
+* [CVE-2021-23424] Upgrades webpack-dev-server and webpack-cli ([#1229](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1229))
+* [CVE] Upgrades hapi from v17 to v20 ([#1146](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1146))
+* [CVE-2022-21670] Bump markdown-it from 10.0.0 to 12.3.2 ([#1140](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1140))
+* [CVE-2022-0155] Removes deprecated request and @percy/agent ([#1113](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1113))
+* [CVE-2021-23490] Bump parse-link-header from 1.0.1 to 2.0.0 ([#1108](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1108))
+* [CVE-2021-3765] Bumps @microsoft/api-documenter and @microsoft/api-extractor ([#1106](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1106))
+* [CVE] Removes KUI Generator and related dependencies ([#1105](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1105))
+* [CVE] Upgrades babel, storybook, and postcss ([#1104](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1104))
+* [CVE] Upgrade immer from 8.0.1 to 9.0.6 ([#780](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/780))
+
+### Features/Enhancements
+* Adds src/test_utils/ and src/core paths to the target clean patterns ([#1442](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1442))
+* Adds code coverage and GitHub Actions badges ([#1498](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1498))
+* Adds code coverage reporting ([#1478](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1478))
+* [Feature] Expressions example plugin ([#1438](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1438))
+* [Circuit-Breaker] Add memory circuit breaker configuration ([#1347](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1347))
+* Fixes search usage telemetry ([#1427](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1427))
```

Comment:
To me, this is a breaking change. For all versions, `1.x` this feature was enabled by default. But for `2.x` we disable it by default and end users will need to opt-in by setting the config to true.

Reply:
Will move this to breaking change